repository/src/main/java/tech/ydb/yoj/repository/db/TableDescriptor.java

```diff
@@ -8,6 +8,10 @@ public static <E extends Entity<E>> TableDescriptor<E> from(EntitySchema<E> sche
         return new TableDescriptor<>(schema.getType(), schema.getName());
     }
 
+    public static <E extends Entity<E>> TableDescriptor<E> from(EntitySchema<E> schema, String name) {
```

Comment:
You already can do it from default constructor. I think it's better to remove this method, especially it is used only one time